setup.sh

```diff
@@ -77,7 +77,15 @@ function updateenv() {
         fi
     fi
 
-    ${PYTHON} -m pip install --upgrade -r ${REQUIREMENTS} ${REQUIREMENTS_HYPEROPT} ${REQUIREMENTS_PLOT}
+    REQUIREMENTS_FREQAI=""
```

Comment:
Note to self:
we'll need to rework this potentially - if "develop" is selected, all other questions are pointless as that installs hyperopt, plotting AND freqAI dependencies.